scripts/reana

```diff
@@ -26,6 +26,8 @@ reana-workflow-monitor
 reana-message-broker
 reana-workflow-engine-yadage"
 
+experiment_list=$(find ./configuration-manifests/namespaces/ -name \*.yaml -printf '%f\n' |  sed -e 's/\.yaml$//')
```

Comment:
`-printf` is not supported on BSD, something like:
`find ./configuration-manifests/namespaces/ -name \*.yaml -type f -exec basename {} .yaml ";"`
should be more portable